lib/runner/test-runner/test-runner.js

```diff
@@ -20,13 +20,19 @@ module.exports = testRunner = {
   _testsRemaining: null,
   _testsToRerun: [],
```

Comment:
Double check but a quick CTRL-F on this variable in the file has 1 occurrence. Can we delete this?

Reply:
Just do a quick check in general if any of these variables are used only once of they are not used correctly now that rerun logic has changed.